ios/PacketTunnel/WireguardDevice.swift

```diff
@@ -318,11 +295,10 @@ class WireguardDevice {
 
             // Re-resolve endpoints on network changes
             if let currentConfiguration = self.configuration {
-                let newResolvedConfiguration = Self.resolveConfiguration(currentConfiguration)
-                let commands = newResolvedConfiguration.endpointUapiConfiguration()
+                let resolvedConfiguration = Self.resolveConfiguration(currentConfiguration)
+                let commands = resolvedConfiguration.endpointUapiConfiguration()
 
                 Self.setWireguardConfig(handle: handle, commands: commands)
-                self.resolvedConfiguration = newResolvedConfiguration
```

Comment:
We never really used `resolvedConfiguration` for anything so I decided to drop it.